setup.py

```diff
@@ -3,7 +3,7 @@
 from setuptools import setup
 
 package_data = {

```

Comment:
If this indeed works and get approved, you could simply remove this assignment
altogether instead of commenting it out. @jjgalvez